contracts/Dandy.cdc

```diff
@@ -2,12 +2,15 @@ import NonFungibleToken from "../contracts/standard/NonFungibleToken.cdc"
 import FungibleToken from "../contracts/standard/FungibleToken.cdc"
 import MetadataViews from "../contracts/standard/MetadataViews.cdc"
 import FindViews from "../contracts/FindViews.cdc"
+import FIND from "../contracts/FIND.cdc"
```

Comment:
I think we need to put MinterPlatform in its own contract. Importing FIND here does not look like a good idea.